bluechi.spec.in

```diff
@@ -127,19 +127,28 @@ SELinux policy associated with the bluechi and bluechi-agent daemons
 %{_mandir}/man8/bluechi*selinux.*
 
 %post selinux
+# Remove hirte policy
+if [ $1 -eq 1 ]; then
+semanage port -N -d -p udp 842 2>/dev/null || true
```

Comment:
looks like is missing a space in these semanage commands?

Reply:
@dougsland I don't see a missing space?

@mwperina bluchi.pp gets installed before hirte is uninstalled.

Reply:
Ah, OK. I missed that, thanks